data/Warm_Cold.py

```diff
@@ -0,0 +1,17 @@
+import random
+
+number = random.randint(1, 50)
+value = input("Please enter a number:\n")
+counter = 0
+while True:
+    counter += 1
+    if number == int(value):
+        print('you won')
+        break
+    elif number < int(value):
+        print('less')
+        value = input("Please enter a number:\n")
```

Comment:
Nie musisz pisać znów wymagać podania liczby, zamień input na continue i zobacz jaki to da efekt :D

Reply:
Przerobione